.gitlab-ci.yml

```
@@ -18,24 +28,24 @@ before_script:
 validate_doc:
   stage: test
   script:
-    - npm exec -- bump deploy --dry-run bump.openapi.v3.yml
+    - npm exec -- bump deploy --dry-run 'bump.openapi.v3.yml' --doc documentation_slug --token bump_token
```

why do you use single quotes around filename and not around the other values? It feels strange to have a mixed style.

Also I'd recommend to also name the doc slug with a mention to bump:
```suggestion
    - npm exec -- bump deploy --dry-run bump.openapi.v3.yml --doc bump_documentation_slug --token bump_token
```

---

About single vs double quote, I'll double check, but I've seen some crash  on [this job](https://gitlab.com/Polo2/restaurant-api/-/jobs/7061347968) , fixed by single-quote.

About `bump_documentation_slug`, fully agreed  👍

---

EDIT:

I confirm that script works with double-quotes too [(cf this job)](https://gitlab.com/Polo2/restaurant-api/-/jobs/7069346543).
what is important is to add quotes, single or double.